docs/README.md

```diff
@@ -110,67 +110,75 @@ python3 runbenchmark.py --help
 
 ```text
 usage: runbenchmark.py [-h] [-m {local,aws,docker,singularity}]
-                       [-t [task_id [task_id ...]]]
-                       [-f [fold_num [fold_num ...]]] [-i input_dir]
+                       [-t [task_id [task_id ...]]
+                       [-f [fold_num ...]] [-i input_dir]
                        [-o output_dir] [-u user_dir] [-p parallel_jobs]
                        [-s {auto,skip,force,only}] [-k [true|false]]
+                       [-e] [--logging LOGGING]
+                       [--openml-run-tag OPENML_RUN_TAG]
                        framework [benchmark] [constraint]
 
 positional arguments:
-  framework             The framework to evaluate as defined by default in
-                        resources/frameworks.yaml. To use a labelled framework
-                        (i.e. a framework defined in
-                        resources/frameworks_{label}.yaml), use the syntax
-                        {framework}:{label}.
-  benchmark             The benchmark type to run as defined by default in
-                        resources/benchmarks/{benchmark}.yaml, a path to a
-                        benchmark description file, or an openml suite or
-                        task. OpenML references should be formatted as
-                        'openml/s/X' and 'openml/t/Y', for studies and tasks
-                        respectively. Use 'test.openml/s/X' for the OpenML test
-                        server. Defaults to `test`.
-  constraint            The constraint definition to use as defined by default
-                        in resources/constraints.yaml. Defaults to `test`.
+  framework             The framework to evaluate as defined by default in resources/frameworks.yaml.
+                        To use a labelled framework (i.e. a framework defined in resources/frameworks-{label}.yaml),
+                        use the syntax {framework}:{label}.
+  benchmark             The benchmark type to run as defined by default in resources/benchmarks/{benchmark}.yaml,
```

Comment:
Are there any reasons that you changed line breaks here?

Reply:
No. I believe I simply copied the new `-h` output into the document, and on my system the line breaks were different than that of the original docs. Sorry for the inconvenience :/